target/linux/realtek/dts-5.10/rtl8380_d-link_dgs-1210-10p.dts

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl838x_d-link_dgs-1210_soc.dtsi"
+
+/ {
+	compatible = "d-link,dgs-1210-10p", "realtek,rtl838x-soc";
+	model = "D-Link DGS-1210-10P";
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		/* is this pin 30 on the external RTL8231 (&gpio1)? */
```

Comment:
left-over debug commit?

Reply:
@oliv3r @musashino205 Do we talk about line 13? But this sounds like a problem in the current -10p device dts. My patch doesn't fix old problems in the current dts. It only makes a new device tree that works with every device based on the current facts. So the old input will be the new output, nothing more or nothing less.

Reply:
This commit has been removed from the PR.